bin/bintray_garbage_collector.rb

```diff
@@ -1,13 +1,31 @@
 require 'dotenv'
+require 'optparse'
+require 'psych'
 require_relative '../lib/garbage_collector.rb'
 
-# We might want to extract this in a file to make it easier to edit
-projects = [
-  { bintray: 'wordpress-mobile/maven/utils', github: 'wordpress-mobile/WordPress-Utils-Android' }
-]
+source = Psych.load(File.read('projects.yml'), symbolize_names: true)
```

Comment:
Ruby has its own YAML loader class, but psych offers this method to convert string keys to symbols which also handles nested arrays. 

It begs the question, why not using string keys? Good question... I don't like them 😳 Since we already had the `Gemfile` setup, adding another gem was cheap. 🤷‍♂️ 

I won't argue with you if you think it's unnecessary overload.